dart/test/test_utils.dart

```diff
@@ -18,6 +18,9 @@ const String _testDsnWithPath =
 const String _testDsnWithPort =
     'https://public:secret@sentry.example.com:8888/1';
 
+/// helper to detect a browser context
+const isWeb = identical(0, 0.0);
```

Comment:
can we use `platformChecker.isWeb`?

Reply:
This is for tests. For example the tests which check for the name of the sdk (`sdkName(bool isWeb)`) don't know about SentryOptions or PlatformChecker. I mean we can read it from PlatformChecker even in test, what do you think?

Reply:
tbh really up to you, its just a nit, I'm not a big fan of DRY.
if we have this code being executed somewhere, I'd not duplicate it and rather reuse it, unless using `platformChecker.isWeb` would require too much refactoring/abstraction, not sure if its the case here